.coderabbit.yaml

```
@@ -0,0 +1,112 @@
+  high_level_summary_placeholder: '@coderabbitai 요약'
+  auto_title_placeholder: '@coderabbitai'
+  poem: true
+  review_status: true
+  collapse_walkthrough: false
+  path_filters:
+    - "!**/README.md"
+  path_instructions:
+    - path: "**"
+      instructions: >
+        Review the API endpoints based on this guidelines.
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)